src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -7,33 +7,43 @@ const dataToChartJSformat = data => data.map(({ datetime, value }) => ({
   y: value,
 }));
 
-const generateInitialOptions = (meta) => {
-  const higherIsBetter = (meta.lower_is_better === false);
+const generateInitialOptions = (series) => {
+  const higherIsBetter = (series.meta.lower_is_better === false);
+  const higherOrLower = higherIsBetter ? 'higher is better' : 'lower is better';
   return {
     reverse: higherIsBetter,
     scaleLabel: higherIsBetter ? 'Score' : 'Load time',
     tooltips: {
       callbacks: {
         footer: (tooltipItems, data) => {
+          const tooltipData = [];
           let delta = 'n/a';
+          let deltaPercentage = 'n/a';
           if (tooltipItems[0].index > 0) {
-            delta = (data.datasets[tooltipItems[0].datasetIndex].data[tooltipItems[0].index].y
-              - data.datasets[tooltipItems[0].datasetIndex].data[tooltipItems[0].index - 1].y)
-              .toFixed(2);
-            }
-            return `Delta: ${delta}`;
-          },
+            const currentData = data.datasets[tooltipItems[0].datasetIndex]
+              .data[tooltipItems[0].index].y;
+            const previousData = data.datasets[tooltipItems[0].datasetIndex]
+              .data[tooltipItems[0].index - 1].y;
+
+            delta = (currentData - previousData).toFixed(2);
+            deltaPercentage = (((currentData - previousData) / previousData) * 100).toFixed(2);
+          }
+          const indicator = `${data.datasets[tooltipItems[0].datasetIndex].data[tooltipItems[0].index].y} (${higherOrLower})`;
+          delta = `Δ ${delta} (${deltaPercentage}%)`;
```

Comment:
This variable doesn't need to be overwritten; it can be passed directly to the `push` call:

```js
tooltipData.push(indicator, `Δ ${delta} (${deltaPercentage}%)`);
```